client/src/store/new-request-module.ts

```diff
@@ -2227,7 +2227,6 @@ export class NewRequestModule extends VuexModule {
 
       if (response?.data) {
         this.mutateNameRequest(response.data)
-        this.mutateDisplayedComponent('Success')
```

Comment:
I removed this from this "network layer" and added it to the code that called this method.